scripts/install_third_party_libs.py

```diff
@@ -308,6 +378,8 @@ def main() -> None:
     # Compile protobuf files.
     python_utils.PRINT('Installing buf and protoc binary.')
     install_buf_and_protoc()
+    rewrite_android_proto_files()
+    move_all_proto_files_to_third_party()
```

Comment:
@DubeySandeep I had introduced these two functions:
1. `rewrite_android_proto_files()` - This function will help us in rewriting the proto files which are coming from the `oppia-proto-api` repository. 
Q - Why do we need this function?
Ans - As per the function https://github.com/oppia/oppia/blob/1fddf2533bcb7aa1a5fa83e8c7d2897f407c200a/scripts/install_third_party_libs.py#L145 , we need to manually configure the import statement https://github.com/oppia/oppia/blob/1fddf2533bcb7aa1a5fa83e8c7d2897f407c200a/scripts/install_third_party_libs.py#L170 , similarly, the proto files which are coming from the `oppia-proto-api` repository consist in multiple subdirectories which makes raise some issue when we use the proto files in the codebase. 
2. `move_all_proto_files_to_third_party()` - This function will move all the proto files inside the folder created by `oppia-proto-api` from any number of subdirectories directly to the `third_party` folder, which will eventually help our `protoc` command make the compiled files usable. 

For more detail - https://github.com/oppia/oppia/pull/13391/#issuecomment-958599772

If you think this is going in the correct direction, could you help me out in what test case should I write for these newly introduced two functions.